docs/console/project-configuration/manage-environment-variables/manage-environment-variables-with-akv.md

```diff
@@ -0,0 +1,99 @@
+---
+id: manage-environment-variables-with-akv
+title: Manage Environment Variables with Azure Key Vault
+sidebar_label: Manage Environment Variables with Azure Key Vault
+---
+
+[Azure Key Vault](https://azure.microsoft.com/en-us/products/key-vault), serves as a valuable solution for securely storing and managing a wide range of sensitive information. This includes not only fundamental key-value pairs but also digital certificates and private keys. It ensures the continuous availability of these resources for our deployments and applications, enhancing security and accessibility.
+
+:::warning
+The current version of the Console can only acquire key-value pairs (aka secrets) from Key Vault.
+:::
+
+You can directly manage the secrets stored in Key Vault using the web interface of your Key Vault instance. The Console's exclusive role is to retrieve these secrets during deployment and subsequently supply them to your runtime environment in the format of Kubernetes Secrets.
+
+<!-- The management of the secrets stored in Key Vault is done directly from the web interface of your Key Vault instance, the Console is responsible for solely fetching the secrets at deploy time and providing them to your runtime environment in the form of Kubernetes Secrets. -->
+
+To correctly configure a Key Vault in your project follow these steps:
+
+1. [Install the `external-secrets` operator](https://external-secrets.io/) in the Kubernetes cluster where your project will be deployed
+2. [Setup the Key Vault instance for the operator authentication](#setup-the-key-vault-instance)
+3. [Setup the project to use the Azure Key Vault provider as a secret manager](#project-configuration)
+
+## Setup the Key Vault instance
+
+There are several ways to enable the operator authentication against the Key Vault instance, the one we chose is the Workload Identity authentication. This authentication method does not require sharing any secret between the cluster and Key Vault.
+
+### Workload Identity setup
+
+The [Workload Identity](https://azure.github.io/azure-workload-identity/docs/) authentication works by establishing a trust relationship between a service account from the Kubernetes cluster's namespace where your project will be deployed and the Azure Active Directory service that is responsible for the authentication and authorization against the Key Vault instance.
+
+#### 1. Create an Azure Active Directory application
+
+First off, you need to create an Azure application aka Service Principal. This application will be used as a conduit to Azure Key Vault and will set up all the required permissions and credentials. Follow the steps reported [here](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#register-an-application-with-microsoft-entra-id-and-create-a-service-principal).
```

Comment:
```suggestion
First off, you need to create an Azure application aka Service Principal. This application will be used as a conduit to Azure Key Vault and will setup all the required permissions and credentials. Follow the steps reported [here](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#register-an-application-with-microsoft-entra-id-and-create-a-service-principal).
```

Reply:
Here `set up` is used as a verb and not a noun, hence the correct one is the two words form